_posts/2022-09-21-simple-angular-dialogs-with-the-cdk.md

```diff
@@ -10,29 +10,27 @@ subclass: 'post'
-to routes as pages. In the previous article we see different approaches of handling dialogs and we focus on the benefits of putting dialogs behind routes.
+to routes as pages. In the previous article, we see different approaches of handling dialogs and we focus on the benefits of putting dialogs behind routes.
 
 ### Some context
 
-We will continue from the context of that [Angular routed dialogs](https://blog.brecht.io/routed-angular-dialogs/) article:
-We have a `UsersComponent` that will display a list of users, and a `UsersDetailComponent` that will display
-the details of that use. When clicking in the list of users on a specific user we want to open its details. The `UsersDetailComponent` wants to use our custom component called `MyDialogComponent` to render the 
-details of a specific user in a nice dialog. We use the same setup as the [Angular routed dialogs](https://blog.brecht.io/routed-angular-dialogs/) article
-so if you didn't read that one yet, you might want to read that first.
-This article focusses on the actual dialog, not on the application structure itself.
-In this article we are going to focus on how we can tackle real dialog functionality with the Angular CDK.
+We will continue from the context of the [Angular routed dialogs](https://blog.brecht.io/routed-angular-dialogs/) article. We have a `UsersComponent` that will display a list of users and a `UsersDetailComponent` that will display
+the details of that use. When clicking in the list of users on a specific user we want to open its details. The `UsersDetailComponent` wants to use our custom component called `MyDialogComponent` to render the details of a specific user, in a nice dialog. We use the same setup as the [Angular routed dialogs](https://blog.brecht.io/routed-angular-dialogs/) article so if you didn't read that one yet, you might want to read that first before reading further.
+
+In this article, we are going to focus on how we can tackle real dialog functionality with the Angular CDK.
 
 ## Why the Angular CDK?
 
-This library provided by Angular Material is focussed on behavior.
-It can provide us with 2 things that we can use for creating modals:
-- The **portal**: This is a piece of UI that can be dynamically rendered to an open slot on the page
-- The **overlay**: This is something we can use to create dialog behavior: It supports position strategies, we can configure a backdrop, and
-it has some basic functionalities that we can leverage for our modal that we don't want to write ourselves. 
+The Angular CDK library is focused on "behavior(s)" that web application needs. In this library are things like: accessibility, Coercion, Drag and Drop etc. And a other advantage is that is maintained by the Angular Material team.
+
+The CDK provides us with 2 things that we can use for creating modals:
+
+1. The [**portal**](https://material.angular.io/cdk/portal/overview): This is a piece of UI that can be dynamically rendered to an open slot on the page
```

Comment:
I would try to simplify the 2 elements with:

```
The [**portal**](https://material.angular.io/cdk/portal/overview), a piece of UI that can be dynamically rendered to an open slot on the page
```

and

```
The [**overlay**](https://material.angular.io/cdk/overlay/overview) to create dialog behavior: It supports position strategies and backdrop behavior and provides some additional basic functionalities to leverage our modal.
```

Reply:
good point thx